src/components/UserManagement/UserAccount/UserAccount.module.scss

```diff
@@ -0,0 +1,164 @@
+@import "styles/variablesscss";
+
+.UserAccount {
+  padding: 62px 100px 162px;
+  font-family: "IBM Plex Sans", sans-serif;
```

Comment:
I think you can apply this to the UserManagement -> .pageContent class so that it gets applied to all User Management pages, instead of having to add it to every page individually.